backend/server/static/api/meetingbook.yaml

```
@@ -278,4 +520,79 @@ definitions:
           A human readable explanation specific to this occurrence of the
           problem.
         example: Connection to database timed out
-
+  CalendarLink:
+    type: "object"
+    required:
+      - id
+      - valid_until
+    properties:
+      id:
+        type: "string"
+        description: "Unique randomly generated string identifier of the calendar. Consists of digits and latin letters"
+        minLength: 10
+      valid_until:
+        type: "string"
+        format: "date-time"
+        description: "Time of link expiration. The link is not accessible after that time. Defined as Datetime string in ISO 8601 https://en.wikipedia.org/wiki/ISO_8601"
+  CalendarLinks:
+    type: "object"
+    required:
+      - links
+    properties:
+      links:
+        description: "array of shareable links"
+        type: array
+        items:
+          $ref: "#/definitions/CalendarLink"
+  Calendar:
+    type: "object"
+    required:
+      - id
+      - slots
+      - valid_until
+    properties:
+      id:
+        type: "string"
+        description: "Unique randomly generated string identifier of the calendar. Consists of digits and latin letters"
+        minLength: 10
+      valid_until:
+        type: "string"
+        format: "date-time"
+        description: "Time of link expiration. The link is not accessible after that time. Defined as Datetime string in ISO 8601 https://en.wikipedia.org/wiki/ISO_8601"
+      slots:
+        description: "Available time slots"
+        type: array
+        items:
+          $ref: "#/definitions/Slot"
+  BookingSettings:
+    description: "settings of the booking. When the guest sends a booking request, his request is checked against these settings"
+    type: "object"
+    properties:
+      duration:
+        type: "object"
+        description: "Duration settings"
+        schema:
+          $ref: "#/definitions/DurationSettings"
+      start_time:
+        type: "object"
+        description: "Start time settings"
+        schema:
+          $ref: "#/definitions/StartTimeSettings"
+  DurationSettings:
+    description: "settings for the booking duration"
+    type: "object"
+    properties:
+      allowed_values:
+        description: "list of allowed time values"
+        type: array
+        items:
+          type: int
+  StartTimeSettings:
+    description: "settings for the booking start time"
+    type: "object"
+    properties:
+      allowed_values:
+        description: "list of allowed values (amount of minutes for each hour). For example values of [15, 45] means that the user can book only time slots starting at XX:15 and XX:45, but not at XX:30."
+        type: array
+        it<ems:
+          type: int
```

```suggestion
          type: integer
```

---

For date-tame we should specify type of items: string
```suggestion
          type: string
```

---

For example values of [15, 45]
it's not datetime

---

Ack